src/extensions/extensionsApi.ts

```diff
@@ -330,12 +347,14 @@ export async function configureInstance(
  * @param instanceId the id of the instance to configure
  * @param extensionSource the source for the version of the extension to update to
  * @param params params to configure the extension instance
+ * @param validateOnly if true, only validates the update and makes no changes
  */
 export async function updateInstance(
   projectId: string,
   instanceId: string,
   extensionSource: ExtensionSource,
-  params?: { [option: string]: string }
+  params?: { [option: string]: string },
+  validateOnly: boolean = false
```

Comment:
Wonder if we should make this signature an interface? The number of arguments for {create|update|delete|configure}Instance seems to be growing.